src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -957,7 +950,7 @@ int getNumDeclaredCandidates() {
   }
 
   int getNumCandidates() {
-    return getCandidateCodeList().size() - excludedCandidates.size();
+    return candidateNames.size() - excludedCandidates.size();
```

Comment:
Same comment as above. Shouldn't we do `getCandidateNames()` here instead?

Reply:
Happy to make these changes, but it feels very strange to me to have `getCandidateNames()` alongside `excludedCandidates`. I could add a `getExcludedCandidates` function, but...we'd be creating a lot of bloat.

I'm used to using getters and setters from outside the class, and accessing members directly within the class. Happy to follow your style here, but do want to flag that there are dozens of direct member accesses throughout this file that aren't getting the same treatment.